portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/ManagementToolbar.testcase

```diff
@@ -145,10 +145,6 @@ definition {
 		}
 
 		task ("Assert the new button is displayed as an icon button") {
-			ScrollWebElementIntoView(locator1 = "ManagementBar#NEW_BUTTON_MOBILE");
-
-			Pause(locator1 = "3000");
-
```

Comment:
It's not failing in [ManagementToolbar#ResponsiveModeNewButtonAsIconButton](https://testray.liferay.com/home/-/testray/cases/1627293472) so I removed it